obconf-qt/obconf-qt_pl.ts

```diff
@@ -771,7 +771,9 @@ Opcje:
         <source>Error while parsing the Openbox configuration file.  Your configuration file is not valid XML.
 
 Message: %1</source>
-        <translation>Błąd wczytania pliku rc.xml. Prawdopodobnie Openbox nie został poprawnie zainstalowany.</translation>
+        <translation>Błąd wczytania pliku rc.xml. Prawdopodobnie Openbox nie został poprawnie zainstalowany.
```

Comment:
IMO "Your configuration file is not valid XML." definitely isn't "Openbox nie został poprawnie zainstalowany."